app/src/main/java/org/jellyfin/mobile/utils/QualityOptions.kt

```diff
@@ -0,0 +1,180 @@
+// Taken and adapted from https://github.com/jellyfin/jellyfin-web/blob/ebb4b050817d4d1920ec28209dee0539410c06ef/src/components/qualityOptions.js
+
+package org.jellyfin.mobile.utils
+
+import org.json.JSONArray
+import org.json.JSONException
+import org.json.JSONObject
+import java.text.DecimalFormat
+import java.text.StringCharacterIterator
+
+object QualityOptions {
```

Comment:
Instead of an object class singleton, I'd prefer this to be injected with our Koin DI.